hnn_core/network.py

```diff
@@ -400,6 +420,51 @@ def __repr__(self):
                  len(self.pos_dict['L5_basket'])))
         return '<%s | %s>' % (class_name, s)
 
+    def _reset_cell_positions(self):
+        pos = _create_cell_coords(n_pyr_x=self._N_pyr_x, n_pyr_y=self._N_pyr_y,
+                                  zdiff=self._layer_separation,
+                                  inplane_distance=self._inplane_distance)
+        for key in pos.keys():
+            self.pos_dict[key] = pos[key]
+
+    @property
+    def inplane_distance(self):
+        return self._inplane_distance
+
+    @inplane_distance.setter
+    def inplane_distance(self, new_dist):
+        if len(self.external_drives) > 0:
+            raise RuntimeError('In-plane distance between network cells can '
+                               'only be adjusted before external drives are '
+                               f'attached; have {len(self.external_drives)} '
+                               'drives.')
```

Comment:
is this still needed now that we moved `inplane_distance` to `parconnect_from_src`?

Reply:
It shouldn't be, but I haven't yet found the bug...